presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -148,6 +150,15 @@ private void updateScreen(WarningsPrinter warningsPrinter)
         printQueryInfo(client.currentStatusInfo(), warningsPrinter);
     }
 
+    private String autoFormatMetricValue(String name, long value)
+    {
+        // TODO: The current implementation of detecting metric type is hacky. If there are more complex use cases, we could consider adding a enum to RuntimeMetric to track its type.
```

Comment:
Let's also mention the rationale of current hacky implementation (e.g. without other usage, it would be too heavy to introduce an enum for metric type).

Reply:
Done.